tests/security_tests.py

```diff
@@ -1122,6 +1132,41 @@ def test_rls_filter_doesnt_alter_energy_query(self):
         assert tbl.get_extra_cache_keys(self.query_obj) == []
         assert "value > 1" not in sql
 
+    @pytest.fixture()
+    def load_unicode_dashboard(self):
```

Comment:
Should we put this in `fixtures.py` so we don't have to repeat this logic wherever we use the unicode dashboard?

Reply:
Sounds good. I just wanted to clarify if you mean moving to `fixtures/` directory? My only concern is about some fixtures that create slices for dashboards (or some other setup). I wondered if wouldn't be better to have just some common parts to combine those dashboard creation parts into particular fixtures (as there is this new file `dashboard_utils`. Or maybe having basic fixture which will be extended or decorated in more complex cases?  WDYT @villebro ?

Reply:
@kkucharc you are right, I meant the `fixtures/` directory! Perhaps we could create something like `fixtures/unicode.py` that would for now just include `def load_unicode_dashboard()`. Down the road I actually think it would be great to try to decouple datasets, charts and dashboards from each other, as may backend tests don't necessarily need the full suite of charts and dashboards, but are ok with just the dataset. Having said that, let's take gradual steps here, so as to not choke on trying to make this perfect on first iteration.

Reply:
That makes sens. I refactored this part. To be honest, I'm proud of this :) - I hope this will be good step on first iteration of this refactor. WDYT?

Reply:
Yeah, I really like this refactor! Hi-5 @kkucharc ! 💯